smtp/client.js

```diff
@@ -85,7 +85,7 @@ Client.prototype =
       if(!(msg instanceof message.Message) 
           && msg.from 
           && (msg.to || msg.cc || msg.bcc)
-          && (msg.text || this._containsInlinedHtml(msg.attachment)))
+          && (msg.text || this._containsInlinedHtml(msg.attachment) || msg.template))
```

Comment:
I think you have an extra parenthesis at the end.

      (msg.text || this._containsInlinedHtml(msg.attachment) || msg.template)